### PR TITLE
Consolidated the recommendation on 20200617

### PR DESCRIPTION
**Pull request by muyuncaigeotab**

### CODE REVIEW

File: src/hardware/developing-an-iox/protocol.md [hunk 1]
```diff
@@ -22,27 +30,30 @@ The Arbitration ID Field for IOX Messages:
...
+| Contents | Reserved: 0 | Message: 0–63 | All IOXs: 0 <br> Individual IOX Destination ID: 1–65535 |
 
 0x1FC00000 IO\_EXPANDER\_RESERVED\_MASK
 
 0x003F0000 IO\_EXPANDER\_COMMAND\_MASK
 
 0x0000FFFF IO\_EXPANDER\_ID\_MASK
 
-### Identifier
+### Destination ID
```
Comment (Stephen-F) [inline src/hardware/developing-an-iox/protocol.md:41]: As you noted, this ID functions as both a destination (when being sent from the GO) or a source (when being sent from an IOX). As such, I think the naming "Destination ID" would likely cause confusion.

Comment (muyuncaigeotab) [reply src/hardware/developing-an-iox/protocol.md:41]: How about changing that to "Address ID"? @jwidz feel free to chime in.

Comment (Stephen-F) [reply src/hardware/developing-an-iox/protocol.md:41]: Address ID sounds fine to me.

Comment (muyuncaigeotab) [reply src/hardware/developing-an-iox/protocol.md:41]: @muyuncai test

Comment (muyuncai) [reply src/hardware/developing-an-iox/protocol.md:41]: I can comment

Comment (muyuncaigeotab) [reply src/hardware/developing-an-iox/protocol.md:41]: Jason couldn't comment, but he's fine with "Address ID" too

**Pull request merged**